vidarr/wdl.py

```diff
@@ -207,6 +239,44 @@ def read_output(output: WDL.Decl):
         'workflow': doc.source_text,
         'accessoryFiles': {
             imported.uri: imported.doc.source_text for imported in doc.imports}}
+    
+    output_meta = doc.workflow.meta.get("output_meta", {})
+
+    # Iterate over each output defined in the workflow
+    for output in doc.workflow.outputs:
+
+        # Check if the output name exists in the output metadata
+        if output.name in output_meta:
+            output_metadata = output_meta[output.name]
+
+            if isinstance(output_metadata, dict) and 'vidarr_label' in output_metadata:
+                vidarr_label = output_metadata['vidarr_label']
+
+                # Define the pattern for searching and replacing the output block
+                pattern = r"(?:workflow)([\s\S]*?)(?:output\s*{)([\s\S]*?)(?=$)"
```

Comment:
`$` matches with the end of the string, which depending on where the output block is, might include the entire rest of the file. `}\n` should match with the end of the output block for all reasonably formatted WDL (let's not worry about someone writing all their WDL on one line lol)